app/controllers/ctc/portal/portal_controller.rb

```diff
@@ -4,10 +4,13 @@ class Ctc::Portal::PortalController < ApplicationController
 
   def home
     if current_client.efile_submissions.any?
-      @status = current_client.efile_submissions.last.current_state
+      submission = current_client.efile_submissions.last
+      @status = submission.current_state
+      @errors = submission.last_transition&.stored_errors
```

Comment:
The `EfileSubmissionTransition#stored_errors` method is cool! :D

Reply:
Yeah, this is the little guy i have some questions about re: whether we should actually persist these so that we can expose rejection reasons to metabase in a more direct manner.